mcs/class/System/Mono.Net.Security/SystemCertificateValidator.cs

```diff
@@ -45,7 +45,8 @@ static SystemCertificateValidator ()
 #elif MONODROID
 			is_macosx = false;
 #else
-			is_macosx = System.IO.File.Exists (OSX509Certificates.SecurityLibrary);
+			// make it work on wine too.
+			is_macosx = Environment.OSVersion.Platform == PlatformID.MacOSX;
```

Comment:
Yeah, that would work in this case, but I think we have _lots_ of places where we make similar assumptions. And I don't understand why calling into the osx security APIs doesn't work in the wine environment (then again, I'm not really familiar with how wine does things).